src/store/flow/block.ts

```diff
@@ -59,11 +60,23 @@ export const mutations: MutationTree<IFlowsState> = {
     currentConfig[key] = value
     findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config = {...currentConfig}
   },
-  block_updateConfigByPath(state, {blockId, path, value}: { blockId: string, path: string, value: object | string }) {
-    set(findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config!, path, value)
-  },
-  block_updateVendorMetadataByPath(state, {blockId, path, value}: { blockId: string, path: string, value: object | string }) {
-    set(findBlockOnActiveFlowWith(blockId, state as unknown as IContext).vendor_metadata!, path, value)
+  block_updateConfigByPath(state, {blockId, path, value}: {blockId: string, path: string, value: object | string}) {
+    // Make nested assignment reactive
+    setWith(
+      findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config!,
+      path,
+      value,
+      (nestedValue, key, nestedObject) => Vue.set(nestedObject, key, nestedValue),
+    )
+  },
+  block_updateVendorMetadataByPath(state, {blockId, path, value}: {blockId: string, path: string, value: object | string}) {
```

Comment:
Just wondering if this mutation is being used anywhere?   I see that you removed it from ContactPropertyEditor

Reply:
thanks, I removed this too